docs/HowTo/Troubleshoot/Use-EVM-Tool.md

```diff
@@ -55,7 +55,25 @@ docker run -rm <docker options> hyperledger/besu-evmtool:develop <evmtool option
 The first mode of the EVM tool runs an arbitrary EVM and is invoked without an extra command.  [Command
 line options](../../Reference/Evm-Tool.md) specify the code and other contextual information.
 
-The EVM tool also has a [`state-test` sub command](../../Reference/Evm-Tool.md#evm-tool-state-test-options)
-that allows the Ethereum State Tests to be evaluated. Most of the options from EVM execution do not apply.
+The EVM tool also has a [`state-test` subcommand](../../Reference/Evm-Tool.md#state-test-options)
+that allows the [Ethereum State Tests](https://github.com/ethereum/tests/tree/e102032ae30ef8ae0f0ec94b9cf37ac53c940cb2/GeneralStateTests) to be evaluated.
```

Comment:
Do we want to link to a specific SHA within the reference tests or just link to the main branch?  e.g. https://github.com/ethereum/tests/tree/develop/GeneralStateTests 

I can see an argument for either, just want to ensure this is intentional